extensions/azurePublish/src/services/appRegistrationService.ts

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { throwNotImplementedError } from './throwNotImplementedError';
+
+/* Use Graph Client
+https://www.npmjs.com/package/@microsoft/microsoft-graph-client
+*/
+export const createAppRegistrationService = (graphToken: string) => {
```

Comment:
Consider if registration services should have different verbs for their methods (register, unregister) than provisioning services (create, delete).